packages/fxa-shared/index.ts

```diff
@@ -29,6 +30,7 @@ module.exports = {
   experiments: {
     BaseGroupingRule,
   },
+  express,
```

Comment:
This seems to import and export `express` but is anything currently using the exported `express` instance?

I found these three instances that seem to be calling `import express from 'express';`:

- `packages/fxa-shared/express/geo-locate.ts`
- `packages/fxa-shared/express/remote-address.ts`
- `packages/fxa-shared/express/routing.ts`

Reply:
```
% rg -l 'fxa-shared/express' | rg -v 'test'
packages/fxa-content-server/server/lib/flow-event.js
packages/fxa-content-server/server/lib/logging/route_logging.js
packages/fxa-content-server/server/lib/routes/get-metrics-flow.js
packages/fxa-payments-server/server/lib/logging/route-logging.js
packages/fxa-payments-server/server/lib/server.js
```